src/main.rs

```diff
@@ -159,10 +160,24 @@ fn main() {
                 .value_name("FILE")
                 .help("Export the timing results as a Markdown table to the given FILE."),
         )
+        .arg(
+            Arg::with_name("compare")
```

Comment:
I believe the `--compare` option will be so useful that we want to enable this by default. What do you think?

*If* we enable this by default, I think we don't even need an option to disable it.. I can't imagine any scenario where a user would not want to see that summary(?).

Reply:
Sure, it makes sense as part of the default application output